intro-HPC/title.tex

```diff
@@ -97,14 +92,39 @@
 \multicolumn{3}{|c|}{\strong{Advanced Part}} \\ \hline
 \strong{Questions}                           & \strong{chapter} & \strong{title} \\ \hline
-Can I compile my software on the \hpc?       & \strong{\ref{ch:compiling-and-testing-your-software-on-the-hpc}}      & \nameref{ch:compiling-and-testing-your-software-on-the-hpc} \\ \hline
 What are the optimal Job Specifications?     & \strong{\ref{ch:fine-tuning-job-specifications}}      & \nameref{ch:fine-tuning-job-specifications} \\ \hline
-%How do I check my programs at runtime?       & \strong{\ref{ch:monitoring-cluster-utilization}}      & \nameref{ch:monitoring-cluster-utilization} \\ \hline
-%Can I stop my program and continue later on? & \strong{\ref{ch:checkpointing}}      & Checkpointing \\ \hline
-Do you have more examples for me?            & \strong{\ref{ch:program-examples}}      & \nameref{ch:program-examples} \\ \hline
-Any more advice?                             & \strong{\ref{ch:best-practices}}      & \nameref{ch:best-practices} \\ \hline
+Can I start many jobs at once?                                          & \strong{\ref{ch:multi-job-submission}} & \nameref{ch:multi-job-submission} \\ \hline
+Can I compile my software on the \hpc?       & \strong{\ref{ch:compiling-and-testing-your-software-on-the-hpc}}      & \nameref{ch:compiling-and-testing-your-software-on-the-hpc} %\\ \hline
+\ifthenelse{\boolean{gent} \OR \boolean{brussel}}{%
+\\ \hline
+Can I stop my program and continue later on? & \strong{\ref{ch:checkpointing}}     & \nameref{ch:checkpointing}
+}{} 
+\\ \hline
+Do you have more program examples for me?     & \strong{\ref{ch:program-examples}}  & \nameref{ch:program-examples} \\ \hline
+Do you have more job script examples for me? & \strong{\ref{ch:jobscript-examples}} & \nameref{ch:jobscript-examples} \\ \hline
+Any more advice?                             & \strong{\ref{ch:best-practices}}    & \nameref{ch:best-practices}  %\\ \hline
+\ifthenelse{\not \boolean{antwerpen}}{%
+\\ \hline
+Need a remote display?                       & \strong{\ref{ch:vnc}}               & \nameref{ch:vnc} 
+}{}
+\\ \hline
 \end{tabular}
 
+\ifgent
+The \strong{Software-specific Best Practices Part} focuses on specific programs.
+
+\begin{tabular}{|p{\dimexpr 0.44\textwidth-2\tabcolsep}|>{\centering\arraybackslash}p{\dimexpr 0.12\textwidth-2\tabcolsep}|p{\dimexpr 0.44\textwidth-2\tabcolsep}|} \hline
+\multicolumn{3}{|c|}{\strong{Software-specific Best Practices Part}} \\ \hline
+MATLAB & \strong{\ref{ch:matlab}} & \nameref{ch:matlab} \\ \hline
+OpenFOAM & \strong{\ref{ch:openfoam}} & \nameref{ch:openfoam} \\ \hline
+Mympirun & \strong{\ref{ch:mympirun}} & \nameref{ch:mympirun} \\ \hline
+Singularity & \strong{\ref{ch:singularity}} & \nameref{ch:singularity} \\ \hline
+SCOOP & \strong{\ref{ch:scoop}} & \nameref{ch:scoop} \\ \hline
+Easybuild & \strong{\ref{ch:easybuild}} & \nameref{ch:easybuild} \\ \hline
+HOD & \strong{\ref{ch:hod}} & \nameref{ch:hod} \\ \hline
```

Comment:
How are we going to remember to keep this up to date if new chapters are added?

Reply:
I think we should have a consistent overview. grep or whatever could easily check whether the label of every chapter is present in title.tex ...

for i in `grep "\label{ch:" ch*tex | cut -f2- -d: | cut -c7-`; do if [[ `grep $i title.tex | wc -l` -ne 1 ]]; then echo "$i missing"; fi; done

Reply:
Done in https://github.com/hpcuantwerpen/vsc_user_docs/pull/2, please review & merge that PR to update this one.

Reply:
Looks good. Thanks!